usaspending_api/broker/management/commands/fabs_nightly_loader.py

```diff
@@ -1,412 +1,192 @@
-import boto3
 import logging
-import time
 
-from datetime import datetime, timedelta, timezone
-from django.conf import settings
+from datetime import datetime, timezone
 from django.core.management.base import BaseCommand
-from django.db import connections, transaction
-from django.db.models import Count
+from django.db import connections
 
-from usaspending_api.awards.models import TransactionFABS, TransactionNormalized, Award
 from usaspending_api.broker import lookups
-from usaspending_api.broker.helpers import get_business_categories
+from usaspending_api.broker.helpers.delete_fabs_transactions import delete_fabs_transactions
+from usaspending_api.broker.helpers.upsert_fabs_transactions import upsert_fabs_transactions
+from usaspending_api.broker.helpers.last_load_date import update_last_load_date
 from usaspending_api.broker.models import ExternalDataLoadDate
-from usaspending_api.common.helpers.dict_helpers import upper_case_dict_values
-from usaspending_api.common.helpers.etl_helpers import update_c_to_d_linkages
-from usaspending_api.common.helpers.generic_helper import fy, timer
-from usaspending_api.etl.award_helpers import update_awards, update_award_categories
-from usaspending_api.etl.broker_etl_helpers import dictfetchall
-from usaspending_api.etl.management.load_base import load_data_into_model, format_date, create_location
-from usaspending_api.references.models import LegalEntity, Agency
+from usaspending_api.common.helpers.date_helper import cast_datetime_to_naive, datetime_command_line_argument_type
+from usaspending_api.common.helpers.generic_helper import timer
 
 
 logger = logging.getLogger("console")
 
-AWARD_UPDATE_ID_LIST = []
-BATCH_FETCH_SIZE = 25000
+
+SUBMISSION_LOOKBACK_MINUTES = 15
```

Comment:
I'm not sure a 15 minute lookback is as beneficial as originally thought.

Reply:
This, in combination with the changeover to be submission driven, **should** be enough to prevent gaps.  The submission `updated_at` timestamp is much nearer to the end of the transaction than the individual award `updated_at` timestamps were.  If a transaction takes longer than 15 minutes to commit we might have an issue, but _usually_ commits happen very quickly.

Or did I miss your meaning?